independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanArchives.java

```diff
@@ -36,7 +36,7 @@
 
 public final class BeanArchives {
 
-    private static final Logger LOGGER = Logger.getLogger(BeanArchives.class);
+    private static final Logger logger = Logger.getLogger(BeanArchives.class);
```

Comment:
I prefer to use upper case for `static final` constants...

Reply:
The logger reference is not a constant, but a final reference, and should NOT be in uppercase. A constant VALUE should be in uppercase. :wink: 

See https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

Reply:
I had the same discussion with the launcher team back then: https://github.com/fabric8-launcher/launcher-application/pull/263

Reply:
Fair enough. For me anything `static final` feels like a constant and I prefer to use upper case because it's very clear in the code (I'm not going to dive into the immutability discussions ;-). However, I don't think it's a good idea to change this at random. It should be either forced by the formatter or just leave this to the author...

Reply:
I agree it should be forced by the formatter at some point. 

PS: Doing a quick grep I found that `logger` is more common in the codebase than `LOGGER` :)

Reply:
You could consider it a constant so long as the reference is immutable. Which I am not sure about with `Logger`. Either way, if we are already using upper-case in various places throughout the code, I'd just keep it that way ;-)